include/lsst/afw/table/arrays.h

```diff
@@ -125,6 +127,9 @@ class ArrayKey : public FunctorKey<ndarray::Array<T const, 1, 1> >,
     bool operator!=(ArrayKey<T> const& other) const noexcept { return !operator==(other); }
     //@}
 
+    /// Return a hash of this object.
+    std::size_t hash_value() const noexcept { return utils::hashCombine(17, _begin, _size); }
```

Comment:
The magic number `17` has now appeared often enough I'm wondering if it should be a named constant in `utils`, but I suppose that gets into a question about what `seed` ought to be that I've already asked there.

Reply:
I'd actually like to avoid that, for fear that the number will get enshrined as something special. As noted in the `utils` thread, it does make sense to pick different seeds for classes that might get hashed together and would be collision-prone if you did so.